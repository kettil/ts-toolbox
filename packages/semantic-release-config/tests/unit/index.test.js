const config = require('../../src');

describe('...()', () => {
  test('it should work', () => {
    expect(config).toMatchSnapshot();
  });
});
