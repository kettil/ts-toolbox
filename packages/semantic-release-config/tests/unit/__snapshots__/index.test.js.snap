// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`...() it should work 1`] = `
Object {
  "branches": Array [
    "+([0-9])?(.{+([0-9]),x}).x",
    "master",
    "main",
    Object {
      "name": "beta",
      "prerelease": true,
    },
    Object {
      "name": "alpha",
      "prerelease": true,
    },
  ],
  "plugins": Array [
    "@semantic-release/commit-analyzer",
    "@semantic-release/release-notes-generator",
    Array [
      "@semantic-release/changelog",
      Object {
        "changelogFile": "CHANGELOG.md",
        "changelogTitle": "# Changelog",
      },
    ],
    "@semantic-release/npm",
    Array [
      "@semantic-release/git",
      Object {
        "assets": Array [
          "CHANGELOG.md",
          "package.json",
          "package-lock.json",
        ],
      },
    ],
    "@semantic-release/github",
  ],
  "tagFormat": "\${version}",
}
`;
